.gitignore

```diff
@@ -9,6 +9,7 @@ Untitled*.ipynb
 *Copy?.ipynb
 dist
 tmp
+.eslintcache
```

Comment:
ah, thank you. That's been turning up lately and making a nuisance of itself, but I wasn't sure where it came from or what it was, I was just leaving it out of commits and hoping it wasn't actually needed...